### DISCUSSION TITLE
Hi, I wanna collaborate with Elementor OS :)

### DISCUSSION THREAD

**Discussion by leonardobetti** [elementary/triage#221] [Ideas]
Hi all,
I'm a UX/UI designer living in the UK and working for the sports betting industry. I installed Elementary OS on an old iMac and I've been quite happy with the results. Huge respect for all the team 😊
I'd love to collaborate with the project and help out with the design and usability. I have some knowledge of design systems on Figma / Sketch, I also can handle banners/print and basic HTML/CSS. I also found the Elementary OS UI Kit on Figma made by Rajdeep Singha and I think it's a great resource to kick off. How can I help with Elementary as a Designer Do you need any mockups, suggestions for the website, documentation, feedback or anything else? What do you guys need it? This is the first open-source project that I would like to collaborate with. I'm keen to hear from you and join your community. Thanks in advance 🙂

**Comment by danirabbit** [comment]
Hey thanks for your interest! A good place to look for places where developers have requested help from a designer is [issues tagged with "Needs Design"](https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Aelementary+label%3A%22Needs+Design%22&type=issues). Sometimes these need mockups, sometimes just like feedback or discussion on how to implement a feature etc.

You mentioned CSS, our system stylesheet is all SASS and you can see the list of open issues for that here: https://github.com/elementary/stylesheet/issues

The website is also open source and could use a lot of love. That is here: https://github.com/elementary/website

Updating the HIG and/or developing new sections would be amazing. That's something I just haven't had a ton of time for. That's also all open source and available here: https://github.com/elementary/hig/issues . It uses markdown with some special sauce for [GitBook](https://app.gitbook.com/)

Basically anything that you find particularly interesting, let me know and I'll do my best to point you in the right direction! But there's a ton of places to get involved :)

**Comment by danirabbit** [comment]
Gonna close this since it seems resolved :)
